xbmc/guilib/GUIComponent.cpp

```diff
@@ -41,12 +43,18 @@ CGUIComponent::~CGUIComponent()
 void CGUIComponent::Init()
 {
   m_pWindowManager->Initialize();
+  m_stereoscopicsManager->Initialize();
+
+  //! @todo This is something we need to change
```

Comment:
absolutely, because exactly this breaks behaviour.  UnloadSkin unregisteres all targets and LoadSkin registers them.

Reply:
How do we fix this?